QGL/Channels.py

```diff
@@ -195,10 +195,11 @@ class Edge(LogicalChannel):
                                 'cutoff': 2,
                                 'drag_scaling': 0,
                                 'sigma': 5e-9,
-                                'riseFall': 20e-9})
+                                'rise_fall': 20e-9})
     gate_chan = Instance((str, LogicalMarkerChannel))
     frequency = Float(0.0).tag(
         desc='modulation frequency of the channel (can be positive or negative)')
+    cnot_impl = Instance(str)
```

Comment:
Should there be a default implementation?

Reply:
I'm not sure, I don't have a preference here. But somehow I broke the tests by merging from master...